lib/vsc/utils/fancylogger.py

```diff
@@ -740,9 +740,7 @@ def setLogLevel(level):
     """
     if isinstance(level, basestring):
         level = getLevelInt(level)
-
-    logger = getLogger(fname=False, clsname=False)
-
+    logger = logging.getLogger()
```

Comment:
@JensTimmerman What's the point of this, in this PR? It even makes tests fail it seems...